package.json

```diff
@@ -161,13 +161,14 @@
     "react-input-autosize": "^3.0.0",
     "react-is": "^17.0.2",
     "react-markdown": "^6.0.2",
-    "react-redux": "^8.0.1",
+    "react-redux": "^8.0.5",
     "react-router": "5.3.1",
     "react-spring": "^8.0.27",
     "react-tooltip": "^4.2.7",
     "react-transition-group": "^2.9.0",
     "react-window": "^1.8.7",
     "regenerator-runtime": "^0.13.6",
+    "reselect": "4.0.0",
```

Comment:
Should we make this `^4.0.0`?

Reply:
I want this pinned explicitly so that we don't have two versions of reselect. Right now it's pinned to the exact version of reselect that @reduxjs/toolkit depends on.